dtale/views.py

```diff
@@ -172,19 +172,16 @@ class DtaleData(object):
     """
 
     def __init__(self, data_id, url):
+        if data_id != None:
+            data_id = int(data_id)
```

Comment:
data_id = data_id if data_id is None else int(data_id)
```suggestion
        data_id = data_id if data_id is None else int(data_id)
```